src/Server/RestRouterHandlers/RestRouterHandler.h

```diff
@@ -118,9 +118,10 @@ class RestRouterHandler : private boost::noncopyable
     }
 
     String processQuery(
-        const String & query,
-        const Poco::JSON::Object & resp = Poco::JSON::Object(),
-        const std::function<void(Block &&)> & callback = [](Block &&) {}) const;
+        const String & query, const std::function<void(Block &&)> & callback = [](Block &&) {}) const;
```

Comment:
Can we use `std::function<void(Block &&)> callback` as the parameter ?

Reply:
There is no need to remove const in this place, because in executeSelectQuery
In, callbacak is also modified with constant, such as: executeSelectQuery(const String & query, ContextPtr query_context, const std::function<void(Block &&)> & callback, bool internal)